tools/travis/.travis.yml

```diff
@@ -12,12 +12,13 @@ php:
 env:
   global:
     - PATH="$HOME/.composer/vendor/bin:$PATH"
+    - SYMFONY_DEPRECATIONS_HELPER=weak
 
 matrix:
   fast_finish: true
   include:
     - php: 5.3
-      env: COMPOSER_FLAGS="--prefer-lowest"
+      env: COMPOSER_FLAGS="--prefer-lowest --prefer-stable"
```

Comment:
I also tried without `"prefer-stable": true` on my `composer.json` file. Same results.

@core23 Could you please give me a concrete working case of what you are saying?